CharlieBackend.Business/Helpers/JWTGenerator.cs

```diff
@@ -19,15 +18,38 @@ public JwtGenerator(IOptions<AuthOptions> authOptions)
         {
             _authOptions = authOptions.Value;
         }
+        public Dictionary<string, string> ReturnJwtDictionary(AccountDto account)
```

Comment:
Please find a more flexible solution for that. 
Use this link - https://stackoverflow.com/questions/972307/how-to-loop-through-all-enum-values-in-c

Reply:
It think it would be better if you name it GetRoleJwtDictionary(AccountDto account).

Reply:
Done! e818162